pkg/k8sutil/client.go

```diff
@@ -41,8 +43,12 @@ func GetKubernetesVersion(cfg *rest.Config) (*version.Version, error) {
 		return nil, err
 	}
 
-	major, _ := strconv.Atoi(ver.Major)
-	minor, _ := strconv.Atoi(ver.Minor)
+	klog.Infof("Kubernetes version read from server: %v.%v", ver.Major, ver.Minor)
+
+	// Suppress all non digits, version might contain special charcters like, <number>+
+	reg, _ := regexp.Compile("[^0-9]+")
+	major, _ := strconv.Atoi(reg.ReplaceAllString(ver.Major, ""))
+	minor, _ := strconv.Atoi(reg.ReplaceAllString(ver.Minor, ""))
```

Comment:
Error handling is missing.

You can avoid it for the regex with `regexp.MustCompile`, but checking that the string really had numbers seems necessary.

Reply:
This is still open.

Reply:
Oops, sorry. I fixed it now.